src/tools/id-mapping/types/idMappingServerParameters.ts

```diff
@@ -0,0 +1,23 @@
+/* Parameters of a ID mapping job as required by the server */
+import { Namespace } from '../../../shared/types/namespaces';
+
+export type From = string;
+
+export type To = string;
+
+export type IDs = string;
+
+export type ServerParameters = {
+  from: From;
+  to: To;
+  ids: IDs;
+};
+
+// same, because no email was provided before
+export type PublicServerParameters = ServerParameters;
+
+export type IDMappingTarget =
```

Comment:
I guess that list will grow with Dan's PR?

Reply:
Open to suggestions for a better variable name

Reply:
Ah I see - `IDMappingResultsNamespace`? `MappingResultsNamespace`?

Reply:
Changed